web-app/js/portal/details/InfoPanel.js

```diff
@@ -27,27 +28,27 @@ Portal.details.InfoPanel = Ext.extend(Ext.Container, {
 
         var rawAbstract = this.dataCollection.getMetadataRecord().get('abstract');
         var abstract = Ext.util.Format.htmlEncode(rawAbstract);
-
         return String.format(
-            '<h4>{0}</h4>\n{1}' +
-            '<h4>{2}</h4>\n<ul>\n{3}</ul>',
+            '<h3>{0}</h3>\n{1}' +
+            '<hr>\n{2}<h3>{3}</h3>\n<ul>\n{4}</ul>',
             OpenLayers.i18n('abstractTitle'),
             abstract,
-            OpenLayers.i18n('webpageLinksTitle'),
-            this._getHtmlForLinks()
-        );
+            this._getMetadataLink(),
+            OpenLayers.i18n('supplementaryLinksTitle'),
+            this._getHtmlsupplementaryLinks()
+            );
     },
 
-    _getHtmlForLinks: function() {
+    _getHtmlsupplementaryLinks: function() {
```

Comment:
Actually, I think `_getSupplementaryLinkAsHtml()` would be a better name.
And, I think also rename `_getMetadataLink()` to `_getMetadataLinkAsHtml()`, for consistency.